### PR TITLE
Complete overhaul before attaching to domain

### PR DESCRIPTION
**Pull request by alicerunsonfedora**
The site has been dramatically updated with some new features:

## User-facing changes
- A new homepage highlights the downloads and offers a space for other developers to feature their work with AliceOS
- A new features page to describe the features of AliceOS, now with @GanstaKingofSA 's GOBFADU page
- A new Studio landing page before linking to the Discord server that acts as an advertisement
- Links to the new beta docs have been implemented as well
- New background made from Trianglify
- Modified logo from @sr229 to fit the navbar also works with the floating text

## Jekyll-related changes
- More parts of sites have been moved into _includes:
  - `head.html` includes the `<head>` tag
  - `navbar.html` includes the navbar
  - `footer.html` includes the footer made from the `p-strip`
  - `project-placeholder.html` includes the placeholder cards on the homepage
- Download data is soft-linked to the buttons using `release.json`. Depending on readiness, version, or beta build, the links will be generated automatically via JavaScript and jQuery.

### CODE REVIEW

File: _includes/navbar.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+                <a href="https://docs.aliceos.app">Docs</a>
+              </li>
+              <li class="p-navigation__link {% if page.url == '/features/' %} is-selected{% endif %}" role="menuitem">
+                <a href="{{ site.baseurl }}/features/">Features</a>
+              </li>
+        <li class="p-navigation__link" role="menuitem">
+          <a href="https://github.com/TheAngelReturns/aliceos">GitHub</a>
+        </li>
+        <li class="p-navigation__link {% if page.url == '/studio/' %} is-selected{% endif %}" role="menuitem">
+            <a href="{{ site.baseurl }}/studio/">The Studio</a>
```
Comment (sr229) [inline _includes/navbar.html]: you do not need ``{{site.baseurl}}`` on this. Just use `/` As per Jekyll will do that for you

File: features.html [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+                        </div>
+                        <div class = "col-6">
+                                <img src="{{ site.baseurl }}/assets/img/aliceos-privacy.png" alt="Permission Dialog">
+                            </div>
+                </div>
+        </div>
+        <div class="p-strip is-shallow">
+                <div class = "row">
+                        <div class = "col-4">
+                                <img src="{{ site.baseurl }}/assets/img/utilities-terminal.svg" width = "75%" alt="Terminal">
```
Comment (sr229) [inline features.html]: Asset linking should be relative

**Review by sr229 — CHANGES_REQUESTED** [review]

**Comment by alicerunsonfedora** [comment]
Alright, changes should be set

**Review by sr229 — APPROVED** [review]

**Comment by sr229** [comment]
Test this first before deploying

**Pull request merged**